.github/workflows/docker-unified.yml

```diff
@@ -572,9 +568,9 @@ jobs:
         run: ./gradlew :metadata-ingestion:codegen
       - name: Download Base Image
         uses: ishworkh/docker-image-artifact-download@v1
-        if: ${{ needs.setup.outputs.publish != 'true' }}
+        if: ${{ needs.setup.outputs.publish != 'true' && steps.filter.outputs.datahub-ingestion-base == 'true' }}
```

Comment:
not a blocker for this PR, but eventually I'd like to move all of the filtering logic (including `dorny/paths-filter@v2` and these checks) into setup, so that the remainder of the workflow doesn't have any boolean logic